Cargo.toml

```diff
@@ -5,12 +5,18 @@ authors = ["Lonami Exo <totufals@hotmail.com>"]
 edition = "2018"
 
 [dependencies]
-grammers-client = { path = "../grammers/lib/grammers-client" }
-grammers-session = { path = "../grammers/lib/grammers-session" }
+grammers-client = { git = "https://github.com/Lonami/grammers.git", rev = "d91dc82" }
+grammers-session = { git = "https://github.com/Lonami/grammers.git", rev = "d91dc82" }
 reqwest = "0.11.3"
-feed-rs = "0.6.0"
+feed-rs = "1.3.0"
 chrono = "0.4.19"
 tokio = { version = "1.5.0", features = ["full"] }
 log = "0.4.14"
 simple_logger = "1.11.0"
 sqlite = "0.26.0"
+
+# Use vendored openssl. We don't depend on it directly.
+openssl = { version = "0.10.55", features = ["vendored"], optional = true }
+
+[features]
+vendored-openssl = ["openssl"]
```

Comment:
I would prefer if we did not need to do this.

I understand it might be a necessity in some systems.

Perhaps the right solution would be to enable this feature in the `openssl` crate by default on such a system configuration. Or to find why it fails to build.